From 66441a4e8fefda33ec324ea5acb0997a9d7e71d1 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Tue, 17 Jul 2018 09:00:28 +0800
Subject: [PATCH 32/73] arm: dts: add rk3308-evb-v10

Sync from rockchip develop-4.4 for RK_EVB_RK3308_DDR3P1164SD4 V10
board, run in AArch32 mode.

Change-Id: I0f2d4085fe04821d25443b5b8ed11b22882e92fb
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 .../arm/boot/dts/rk3308-evb-dmic-i2s-v10-aarch32.dts | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 arch/arm/boot/dts/rk3308-evb-dmic-i2s-v10-aarch32.dts

diff --git a/arch/arm/boot/dts/rk3308-evb-dmic-i2s-v10-aarch32.dts b/arch/arm/boot/dts/rk3308-evb-dmic-i2s-v10-aarch32.dts
new file mode 100644
index 000000000000..e5377f3cf3c3
--- /dev/null
+++ b/arch/arm/boot/dts/rk3308-evb-dmic-i2s-v10-aarch32.dts
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2018 Fuzhou Rockchip Electronics Co., Ltd
+ */
+
+#include "arm64/rockchip/rk3308-evb-dmic-i2s-v10.dts"
+
+/ {
+	model = "Rockchip RK3308 evb digital-i2s mic board (AArch32)";
+	compatible = "rockchip,rk3308-evb-dmic-i2s-v10-aarch32", "rockchip,rk3308";
+};
+
+&ramoops_mem {
+	reg = <0x0 0x30000 0x0 0x20000>;
+};
+
+&ramoops {
+	record-size = <0x0 0x00000>;
+	console-size = <0x0 0x20000>;
+};
-- 
2.11.0

